ggml/src/ggml-cuda/common.cuh

```diff
@@ -461,6 +470,64 @@ static __device__ __forceinline__ float warp_reduce_max(float x) {
     return x;
 }
 
+static __device__ __forceinline__ unsigned int get_warp_mask() {
```

Comment:
sizeof(uint) is only 4. Maybe we should typedef a ggml_lane_mask_t to uint32_t on cuda and uint64_t on hip.